sdk/containerregistry/container-registry/samples/v1-beta/javascript/README.md

```diff
@@ -5,10 +5,10 @@ languages:
 products:
   - azure
   - azure-container-registry
-urlFragment: container-registry-javascript
+urlFragment: container-registry-javascript-beta
 ---
 
-# Azure Container Registry client library samples for JavaScript
+# Azure Container Registry client library samples for JavaScript (Beta)
```

Comment:
Is this intentional? If you don't want the beta tags and instead want to regenerate GA samples, you can change the version number temporarily, regenerate the samples, and then change the version number back.

Reply:
I was thinking about moving the folder back to `v1` when I actually update the version number, but your suggestion makes sense too.

Reply:
Since the beta generation does more than just change the version number, if you want GA-style samples, you have to generate with a GA version selector. I'm going to create an issue to re-enable the `--override-version` flag, since I could have used it when reworking storage samples as well.

Reply:
yeah I meant to re-generate and re-publish when I change the version number to `1.0.0`

Reply:
@witemple-msft does this look good now?